src/app/Main/Body/QueryPage/Container-Choose.component.tsx

```diff
@@ -13,11 +19,19 @@ export function ContainerChooseComponent(props: {
     dataIsSet: boolean;
 }): JSX.Element {
     if (props.status !== undefined) {
-        if (
-            props.status.isolateStatus === 404 ||
-            props.status.filterStatus === 404
-        ) {
-            return <ErrorPagePageNotFoundComponent />;
+        if (props.status.isolateStatus !== 200) {
+            return (
+                <ErrorPagePageNotFoundComponent
+                    errorStatus={props.status.isolateStatus}
+                />
+            );
+        }
+        if (props.status.filterStatus !== 200) {
+            return (
+                <ErrorPagePageNotFoundComponent
+                    errorStatus={props.status.filterStatus}
+                />
+            );

```

Comment:
You can remove the 200 checks in the next statement now. There are obsolete

Reply:
Done.